data_gemma/vertexai_api.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""VertexAI LLM Interface."""
+
+from data_gemma import base
+from dataclasses import dataclass
+from google.cloud import aiplatform
+import time
+import logging
+
+
+@dataclass(kw_only=True)
+class VertexAIModelConfig:
+  # ID of the GCP project where the model is hosted
+  project_id: str = None
+  # location where the model is hosted
+  location: str = None
+  # prediction endpoint id for the model
```

Comment:
what is a prediction endpoint?

Reply:
this is the id for the endpoint to use to make calls to the model, updated